whos-online/whos-online.js

```diff
@@ -0,0 +1,51 @@
+const whosOnline = (friends) => {
+  
+  friends = [
+     {
+      username: 'Lucy',
+      status: 'offline',
+      lastActivity: 22
+    }, {
+      username: 'Bob',
+      status: 'online',
+      lastActivity: 104
+    }
+  ]
+
+  function userNewStatus(item){
```

Comment:
This function could be outside `whosOnline` because it doesn't depend of `whosOnline`.

Reply:
I rewrote this as a `friends.forEach`, to avoid the use of `map`. 
As for moving it outside of `whosOnline`, if I move it would it get executed in CW?

Reply:
Yes. You don't need to move it outside the file, just outside the function. CW will call the `whosOnline` function which can call other functions.